docs/release/release_notes.md

```diff
@@ -2,17 +2,56 @@
 
 We are pleased to announce the release of DAOS version 2.0.
 
-# DAOS Version 2.0.2 (2022-03-21)
 
-TBD
+# DAOS Version 2.0.2 (2022-03-21)
 
 ## Updates in this Release
 
-TBD
+The DAOS 2.0.2 release contains the following updates on top of DAOS 2.0.1:
+
+- DAOS 2.0.2 includes fixes to the EC, VOS, MS, telemetry, dtx, object components,
+  raft, VMD, as well as the test and build infrastructure.
+
+- The `daos_server` and `daos_agent` daemons now have a secondary group membership
+  in a new `daos_daemons` Linux group
+  [DAOS-6344](https://daosio.atlassian.net/browse/DAOS-6344).
+
+- The sightings with SOAK testing in 2.0.1 have been resolved.
+  See [DAOS-9725](https://daosio.atlassian.net/browse/DAOS-9725).
+
+- Minor changes to the
+  DFS API [DAOS-10009](https://daosio.atlassian.net/browse/DAOS-10009) and 
+  DAOS Java API [DAOS-9379](https://daosio.atlassian.net/browse/DAOS-9379).
+
+- Go dependency has been updated to >= 1.17
+  [DAOS-9908](https://daosio.atlassian.net/browse/DAOS-9908).
+
+- Hadoop dependency has been updated to 3.3.2
+  [DAOS-10068](https://daosio.atlassian.net/browse/DAOS-10068).
+
+- `mercury` has been updated from 2.1.0~rc4-3 to 2.1.0~rc4-5.
+  This fixes multiple issues with `dmg pool delete`, including
+  [DAOS-9725](https://daosio.atlassian.net/browse/DAOS-9561) and
+  [DAOS-9006](https://daosio.atlassian.net/browse/DAOS-9006).
+
+- `spdk` version has been updated from 21.07-11 to 21.07-13 (no content changes).
```

Comment:
Of the notable issues, i would also mention:
- the pool destroy issue
- EC aggregation running too frequently and consuming CPU cycles even if EC isn't used
- disabling the DAOS readahead in douse when write through cache is used (DAOS-9738)